docker/token/token.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+if [ -f /etc/client/client_id ] && [ -f /etc/client/client_secret ]; then
+    export client_id=`cat /etc/client/client_id`
+    export client_secret=`cat /etc/client/client_secret`
+    curl -s -d grant_type=client_credentials -d scope="profile" -u ${client_id}:${client_secret} https://cms-auth.web.cern.ch/token | python2 -c 'import sys,json; print(json.loads(sys.stdin.read())["access_token"])' > /tmp/token
+
+    if [ -f /tmp/token ]; then
+        kubectl create secret generic token-secrets \
```

Comment:
please add echo to indicate when token is created, this will be useful to have in k8s logs when we'll need to debug token issue and will need to know when last time token was successfully created. Please put timestamp and proper message, including size of /tmp/token file.